Makefile

```diff
@@ -131,7 +132,6 @@ env.d/development:
 bootstrap: env.d/development ## Prepare Docker images for the project
 	@$(COMPOSE) build base;
 	@$(COMPOSE) build app;
-	@$(YARN) build-dev
```

Comment:
if you remove this how is the front built?

Reply:
in the previous step, in `build app`